### PR TITLE
add wavefront, scalyr and elastic to output stream config

### PR DESCRIPTION
**Pull request by soyvural**

### CODE REVIEW

**Comment by soyvural** [comment]
@zquiring001 I can not add you as a reviewer, could you please review this PR ?

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```
Comment (yildizfatih) [inline .idea/.gitignore]: let's include .idea folder in .gitignore since it may not play well with gitbook and it can create confusion.

**Review by yildizfatih — APPROVED** [review]
LGTM. thanks.

File: .idea/.gitignore (hunk 1 above)
Comment (soyvural) [reply .idea/.gitignore]: sure, I removed .idea files now only one file is commited

**Comment by zquiring001** [comment]
Looks good to me!

**Pull request merged**